{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--4926d6ca-009c-4169-8ab9-f56f36b43305",
    "created": "2024-08-13T16:14:23.21218Z",
    "modified": "2024-08-13T16:14:23.212185Z",
    "name": "No Title Available",
    "description": "Algorithmic complexity vulnerability in the ssl.match_hostname function in Python 3.2.x, 3.3.x, and earlier, and unspecified versions of python-backports-ssl_match_hostname as used for older Python versions, allows remote attackers to cause a denial of service (CPU consumption) via multiple wildcard characters in the common name in a certificate.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2013-2099"
        }
    ],
    "references": [
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=963260"
        },
        {
            "name": "USN-1984-1",
            "tags": [
                "vendor-advisory",
                "x_refsource_UBUNTU"
            ],
            "url": "http://www.ubuntu.com/usn/USN-1984-1"
        },
        {
            "name": "55116",
            "tags": [
                "third-party-advisory",
                "x_refsource_SECUNIA"
            ],
            "url": "http://secunia.com/advisories/55116"
        },
        {
            "name": "RHSA-2016:1166",
            "tags": [
                "vendor-advisory",
                "x_refsource_REDHAT"
            ],
            "url": "https://access.redhat.com/errata/RHSA-2016:1166"
        },
        {
            "name": "USN-1985-1",
            "tags": [
                "vendor-advisory",
                "x_refsource_UBUNTU"
            ],
            "url": "http://www.ubuntu.com/usn/USN-1985-1"
        },
        {
            "tags": [
                "x_refsource_CONFIRM"
            ],
            "url": "http://bugs.python.org/issue17980"
        },
        {
            "name": "55107",
            "tags": [
                "third-party-advisory",
                "x_refsource_SECUNIA"
            ],
            "url": "http://secunia.com/advisories/55107"
        },
        {
            "name": "RHSA-2014:1690",
            "tags": [
                "vendor-advisory",
                "x_refsource_REDHAT"
            ],
            "url": "http://rhn.redhat.com/errata/RHSA-2014-1690.html"
        },
        {
            "name": "USN-1983-1",
            "tags": [
                "vendor-advisory",
                "x_refsource_UBUNTU"
            ],
            "url": "http://www.ubuntu.com/usn/USN-1983-1"
        },
        {
            "name": "[oss-security] 20130515 Re: CVE Request (minor) -- Python 3.2: DoS when matching  certificate with many '*' wildcard characters {was: CVE Request  (minor) --  python-backports-ssl_match_hostname: Denial of service when matching  certificate with many '*' wildcard characters }",
            "tags": [
                "mailing-list",
                "x_refsource_MLIST"
            ],
            "url": "http://www.openwall.com/lists/oss-security/2013/05/16/6"
        }
    ]
}